sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -215,7 +219,10 @@ export abstract class MessageReceiver extends LinkEntity<Receiver> {
    * You may want to set up retries to recover the broken link and/or report error to user.
    * @param error - The error accompanying the receiver/session error or connection disconnected events
    */
-  abstract onDetached(error?: AmqpError | Error): Promise<void>;
+  abstract onDetached(
+    error?: AmqpError | Error,
+    operationOptions?: OperationOptionsBase
```

Comment:
Are operationOptions optional here? Or are they just allowed to be undefined? (ie, should the caller be forced to opt-out of passing the argument?)

Reply:
Since `A required parameter cannot follow an optional parameter`.
We'd need to make both of them non-optional. Should I do that? (I don't see a problem in doing that.)

Reply:
Done.